brave-kafka/src/main/java/smartthings/brave/kafka/consumers/DefaultSpanNameProvider.java

```diff
@@ -0,0 +1,24 @@
+
+/**
+ * Copyright 2016-2017 SmartThings
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package smartthings.brave.kafka.consumers;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+
+public class DefaultSpanNameProvider<K> implements SpanNameProvider<K> {
+  @Override
+  public String provide(ConsumerRecord<K, byte[]> record) {
+    return "Receive " + record.topic() + "-"  + record.partition();
```

Comment:
I wonder if the partition would be better as an attribute?  Reason would simply be to keep the name displayed in the UI a bit cleaner.  This also seems like something you might only care about when you drill into a particular trace.